src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthControllerTest.java

```diff
@@ -181,10 +197,17 @@ private static Stream<LocalDateTime> nonWorkingHoursScenarioProvider() {
 
     private static Stream<LocalDateTime> workingHoursScenarioProvider() {
         return Stream.of(
-            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 9, 30),
-            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 9, 31),
-            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 18, 30),
-            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 18, 29)
+            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 8, 30),
+            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 8, 31),
+            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 16, 29),
+            LocalDateTime.of(2022, Month.SEPTEMBER, 01, 16, 30)
+        );
+    }
+
+    private static Stream<LocalDateTime> workingHoursWithTimeZoneScenarioProvider() {
+        return Stream.of(
+            LocalDateTime.of(2024, Month.JANUARY, 01, 10, 00),
+            LocalDateTime.of(2024, Month.MAY, 01, 9, 00)
```

Comment:
What about DayLight Saving time? Will this be handled correctly? 
I.e. March 31st 2024, 17:00pm. Will this test pass?

Reply:
I added a new nonWorkingHours edge case Ben.